shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -897,7 +920,9 @@ public static TextDirection fromInt(int value) {
         SemanticsObject parent;
         List<SemanticsObject> childrenInTraversalOrder;
         List<SemanticsObject> childrenInHitTestOrder;
-        List<CustomAccessibilityAction> customAccessibilityAction;
+        List<CustomAccessibilityAction> customAccessibilityActions;
+        CustomAccessibilityAction onTapOverride;
```

Comment:
Technically I think _all_ standard actions can be overriden, but tap and long press are the most common.  I think it is reasonable to start here, since I still don't know what effect the other actions have.